.github/workflows/build-nix.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-14]
+        os: [ubuntu-24.04, macos-14]
```

Comment:
How do we remember to keep up to date? I guess we'll just have to remember to check once a year or something and decide if we want to upgrade?

Speaking of that, we should we switch from macos-14 to macos-15? And maybe from macos-13 to macos-14?

Reply:
Ubuntu does an update to their LTS once every two years, so I don't think keeping up with it is much of a problem.

I think we want to keep macOS 13 because it's the last x86_64 macOS platform. macos-15 is still in Public Preview; 14 is just what the default is, and we're already fairly explicit about that.